andela/travel_tool_back
# #159236855 Setup a blank NodeJS project

#### What does this PR do?
Set up a blank NodeJS project

#### Description of Task to be completed?
- Create package.json file and include scripts
- Set up eslint using airbnb style guide
- Set up babel for transpiling es6 code

#### How should this be manually tested?
- Clone the project
- Run `cd travel_tool_back` to enter the project's directory
- Run `yarn install` to install application dependencies
- Write some javascript code in the `src/index.js` file and run `npm start`
- The code should run properly

#### Any background context you want to provide?
N/A

#### What are the relevant Github Issues?
#159236855

#### Questions:
N/A
